src/internal_modules/roc_audio/latency_tuner.h

```diff
@@ -114,15 +136,23 @@ struct LatencyConfig {
     //!  Negative value is an error.
     float scaling_tolerance;
 
+    //! Latency tuner decides to adjust target latency if
+    //! the current value >= estimated optimal latency * upper_threshold_coef.
+    float upper_threshold_coef;
```

Comment:
From this name it's not clear where this threshold is applied. I guess something like `latency_update_threshold` would be more descriptive.